wiki/Skin/en.md

```diff
@@ -4,7 +4,29 @@ stub: true
 
 # Skin
 
-*See also: [Skinning](/wiki/Skinning)*\
-*For the dedicated sub-forum for skins, see [Skinning](https://osu.ppy.sh/community/forums/15)*
+A **skin** is a set of graphics (images) and custom sample sets (audio clips) that replace the default ones used by osu!. Many players regularly engage in the use of skins to allow for a more customised user experience, should they dislike the default skin. Almost all of what is visible or audible in osu! can be "skinned." 
```

Comment:
> Almost all of what is visible or audible in osu! can be "skinned."

This is incorrect. A very very big chunk of osu is actually unskinnable. Also, the period should come after the closing quotes

Reply:
> > Almost all of what is visible or audible in osu! can be "skinned."
> 
> This is incorrect. A very very big chunk of osu is actually unskinnable. Also, the period should come after the closing quotes

This was the stub's default definition of a skin. So I guess I'll do away with it and write my own understanding and definition